bokehjs/src/coffee/models/glyphs/quad.coffee

```diff
@@ -25,16 +25,10 @@ class QuadView extends Glyph.View
 
     for i in [0...left.length]
       l = left[i]
-      if isNaN(l) or not isFinite(l)
-        continue
       r = right[i]
-      if isNaN(r) or not isFinite(r)
-        continue
       t = top[i]
-      if isNaN(t) or not isFinite(t)
-        continue
       b = bottom[i]
-      if isNaN(b) or not isFinite(b)
+      if isNaN(l+r+t+b) or not isFinite(l+r+t+b)
```

Comment:
here is the line that checks the sum of the values, instead of checking each value individually